### PR TITLE
Load qualifier cards from JSON file; standardize event PDF file names

### PR DESCRIPTION
**Pull request by chlohal**
I thought it might be good to have the qualifier cards be constructed from a JSON file, rather than having manually written data. This fixes the broken links to the team list for completed qualifiers (as they seem functionally the same, I renamed "results" files to "teams" files & set both links to the same URL). This can also provide pre-existing code for interacting with a back-end (assuming there will eventually be one).

### CODE REVIEW

**Comment by broad-well** [comment]
Although the effort to separate data from markup is commendable, keep in mind that there is no need for **every** client to use AJAX additionally to process a JSON file in order to populate the event page. These only change up to twice a week, so this computation is unnecessary and redundant. What _is_ a viable option, however, is to introduce a script that will populate `events.html` with data from the JSON file upon deployment. This will keep the site static and eliminate the need for a separate backend, which limits our options for hosting. It also facilitates event updates for Loretta.

File: script.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+"<label>Qualifier "+E[l].qualNum+"</label>" +
+"<h2>"+E[l].name+"</h2>" +
+'<p class="date">' + A[D.getMonth()] + ' ' + D.getDate() + '<sup>' + (N[D.getDate() % 10] || 'th') + '</sup>' +
+'<br><span class="time">' + E[l].timeRange + '</span>' +
+'<p class="location">' + E[l].location + '</p>' +
+'<p class="region">' + E[l].region + '</p>' +
+'<div class="buttonset"> <a href="events/' + (E[l].results_url||(D.getMonth()<6?D.getFullYear()-1:D.getFullYear()) +'-'+ E[l].qualNum +'-qual-teams.pdf')+'">Teams</a>' +
+'<a href="events/' + (E[l].results_url||(D.getMonth()<6?D.getFullYear()-1:D.getFullYear()) + '-' + E[l].qualNum +'-qual-teams.pdf')+'"'+(D.getTime()>Date.now()?' disabled':'')+'>Results</a> </div>'
+//append the card element to the container
+s.appendChild(f);
```
Comment (broad-well) [inline script.js:37]: Not to mean this personally, but this script is **dreadfully styled**. You have single-letter variable names, no indentations, no spacing, no functions to logically segregate different responsibilities, useless comments that only repeat the code, and inconsistent semicolon placement. Please read [Clean Code](https://www.investigatii.md/uploads/resurse/Clean_Code.pdf).

**Comment by coleh2** [comment]
Ok, thank you for the reference. I agree with you on all of your points, but I'm confused about one-- where do you think I should have added functions?

File: script.js (hunk 1 above)
Comment (chlohal) [reply script.js:37]: Ok, thank you for the reference. I agree with you on all of your points, but I'm confused about one-- where do you think I should have added functions?

Comment (broad-well) [reply script.js:37]: For instance, you could've extracted a function that's meant to render one qualifier based on the given information. This applies to the body of the `for` loop on line 18.

Comment (chlohal) [reply script.js:37]: Alright; thank you!

**Pull request closed without merging**